app/controllers/clearance/sessions_controller.rb

```diff
@@ -33,4 +40,8 @@ def url_after_create
   def url_after_destroy
     sign_in_url
   end
+
+  def url_for_signed_in_users
+    url_after_create
```

Comment:
`url_for_signed_in_users` makes sense to redirect to the `Clearance.configuration.redirect_url`. I think that is the most common configuration behavior. However, I think the reason for this PR is to be able to have two different redirect URLs depending on if you are a newly created user, or a user who has already logged in. Oh wait...maybe I'm reading this wrong. My thought was that I want to be able to specify a different redirect URL for a newly created user (taking them to a "fill out additional profile information" page). But maybe `create` in this context means the creation of a new session.